packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx

```diff
@@ -22,6 +23,7 @@ type BalanceWithCarryoverProps = Omit<
   budgeted?: Binding;
   disabled?: boolean;
   carryoverStyle?: CSSProperties;
```

Comment:
We can remove this prop since caller can just set the style on the `carryoverIndicator`

Reply:
This is currently needed for the modals, they apply some additional style.

Reply:
Those could be updated to use `carryoverIndicator` prop instead. To prevent code duplication, the default arrow component can be exported from the `BalanceWithCarryover` component